### DISCUSSION TITLE
Download location

### DISCUSSION THREAD

**Discussion by pratikga** [icloud-photos-downloader/icloud_photos_downloader#587] [Q&A]
Thank you for making the downloads easy with this great tool!! 

Does this automatically sync up in the photos app or are individual images downloaded in a specific directory? After running the script successfully, I do not see the images in the specified folder.

**Comment by pratikga** [comment]
Edit: I have run this script with docker.

**Comment by AndreyNikiforov** [comment]
stale discussion
